server/src/main/java/org/elasticsearch/cluster/metadata/Metadata.java

```diff
@@ -938,6 +943,24 @@ public Map<String, Custom> customs() {
         return this.customs;
     }
 
+    /**
+     * Returns the full {@link OperatorMetadata} Map for all
+     * operator namespaces.
+     * @return a map of namespace to {@link OperatorMetadata}
+     */
+    public Map<String, OperatorMetadata> operatorMetadata() {
+        return this.operatorMetadata;
+    }
+
+    /**
+     * Returns the {@link OperatorMetadata} for a given namespace.
+     * @param namespace the namespace to lookup operator metadata for
+     * @return {@link OperatorMetadata} or null if not found
+     */
+    public OperatorMetadata operatorMetadata(String namespace) {
```

Comment:
Why have this method and also the one above returning the entire map? If the one above is needed, then it could also be used in place of this method?